.travis.yml

```diff
@@ -5,5 +5,5 @@ jdk:
   - oraclejdk8
 
 sbt_args: -no-colors
-script: sbt ++$TRAVIS_SCALA_VERSION scalastyle clean coverage test unidoc
+script: sbt scalastyle clean coverage test unidoc "project unicorn-core" +test
```

Comment:
no sure how to do it that only unicorn-core is cross-built; in current version it builds:

1) everything with 2.11.x once
2.1) unicorn-core with 2.11.x (actually it doesn't build it, its already built, it only runs tests second time)
2.2) unicorn-core with 2.12.x

Kinda inelegant (but does the job :/